src/translation/translations/en.json

```diff
@@ -515,8 +515,16 @@
     "youAreStaking": "You are staking"
   },
   "vote": {
+    "delegateAddress": "Delegate Address",
+    "delegateLeaderboard": "Delegate Leaderboard",
+    "delegateVoting": "Delegate Voting",
+    "delgateVotes": "Delegate Votes",
+    "enterContactAddress": "Enter contact address",
     "governanceProposals": "Governance proposals",
-    "votingWallet": "Voting wallet"
+    "pasteYourAddress": "Paste your Address",
```

Comment:
I've tried to comment on the designs wherever I found capitalized words in the middle of sentences, but it hasn't been updated everywhere.
The design team agreed we shouldn't capitalize words in the middle of sentences.